src/utils/azureUtils.ts

```diff
@@ -19,4 +21,10 @@ export namespace azureUtils {
     export function getResourceGroupFromId(id: string): string {
         return parseResourceId(id)[2];
     }
+
+    export function getEventGridManagementClient(nodeOrContext: IAzureNode | ISubscriptionWizardContext): EventGridManagementClient {
```

Comment:
Might be worth standardizing methods for getting all of the various clients here.